### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by christoofar** [ivangabriele/clamav-desktop#131] [Questions & Answers] [answered]
Looks promising but not seeing where the roadmap to release cutoff is.

**Comment by ivangabriele** [comment, marked as answer]
Hi, no the project is not dead, just incredibly slow since I'm a single developer (at least until the next release is done).

**Reply by InfoSecUniversity** [reply]
When we will see the next release?

**Comment by Neustradamus** [comment]
@ivangabriele: Please do not stop, your project is very good!

**Comment by Neustradamus** [comment]
Linked to:
- https://github.com/ivangabriele/clamav-desktop/issues/235
- https://github.com/ivangabriele/clamav-desktop/discussions/237
- https://github.com/ivangabriele/clamav-desktop/issues/216
- https://github.com/ivangabriele/clamav-desktop/discussions/239
- https://github.com/ivangabriele/clamav-desktop/issues/211
- https://github.com/ivangabriele/clamav-desktop/discussions/238
- https://github.com/ivangabriele/clamav-desktop/issues/187
- https://github.com/ivangabriele/clamav-desktop/discussions/190
- https://github.com/ivangabriele/clamav-desktop/issues/180
- https://github.com/ivangabriele/clamav-desktop/discussions/188
- https://github.com/ivangabriele/clamav-desktop/issues/127
- https://github.com/ivangabriele/clamav-desktop/discussions/131
- https://github.com/ivangabriele/clamav-desktop/issues/118
- https://github.com/ivangabriele/clamav-desktop/discussions/130
- https://github.com/ivangabriele/clamav-desktop/issues/85
- https://github.com/ivangabriele/clamav-desktop/discussions/87
